cmd/background-newdisks-heal-ops.go

```diff
@@ -451,8 +455,25 @@ func healFreshDisk(ctx context.Context, z *erasureServerPools, endpoint Endpoint
 		return err
 	}
 
-	healingLogEvent(ctx, "Healing of drive '%s' is finished (healed: %d, skipped: %d, failed: %d).", disk, tracker.ItemsHealed, tracker.ItemsSkipped, tracker.ItemsFailed)
+	// if objects have failed healing, we attempt a retry to heal the drive upto 3 times before giving up.
+	if tracker.ItemsFailed > 0 && tracker.RetryAttempts < 4 {
+		tracker.RetryAttempts++
```

Comment:
I think we need:
```
        tracker.update(ctx)
```

because tracker is loaded again in healFreshDisk()

Reply:
It is not persistent on purpose @vadmeste retries a in-memory only tracking.